homeassistant/components/flexit_bacnet/const.py

```diff
@@ -32,3 +32,6 @@
     PRESET_HOME: VENTILATION_MODE_HOME,
     PRESET_BOOST: VENTILATION_MODE_HIGH,
 }
+
+MIN_FAN_SETPOINT = 30
+MAX_FAN_SETPOINT = 100
```

Comment:
These can be local constants inside `number.py`.
```python
_MIN_FAN_SETPOINT = 30
_MAX_FAN_SETPOINT = 100
```

Reply:
Done